moped-editor/src/views/projects/projectView/ProjectView.js

```diff
@@ -164,9 +165,10 @@ const ProjectView = () => {
     : "/moped/projects";
 
   // Get the tab query string value and associated tab index.
-  // If there's no query string, default to first tab in TABS array
-  let activeTabIndex = !!query.get("tab")
-    ? TABS.findIndex((tab) => tab.param === query.get("tab"))
+  // If there's no query string (or the string does not match
```

Comment:
Glad that you addressed this and that I steered away from messing with this code to fix the project component links. Leaning further into React Router `useSearchParams` would have mega broken this! 😵‍💫